api/cookies/cookie_clearer/popup.js

```diff
@@ -0,0 +1,101 @@
+const form = document.getElementById("control-row");
+const go = document.getElementById("go");
+const input = document.getElementById("input");
+const message = document.getElementById("message");
+
+
+document.addEventListener("DOMContentLoaded", initPopupWindow);
+form.addEventListener("submit", handleFormSubmit);
+
+async function initPopupWindow() {
+  let [tab] = await chrome.tabs.query({ active: true, currentWindow: true });
+
+  if (tab.url) {
```

Comment:
Is it possible that `tab` is undefined here?

Reply:
It is technically possible, yes. I'll add tab to the if check to make sure it's not undefined.